core/domain/stats_domain_test.py

```diff
@@ -547,6 +720,106 @@ def test_validate(self):
             'Invalid action type: %s' % 'InvalidActionType')):
             playthrough.validate()
 
+    def test_validate_exp_id(self):
+        playthrough = stats_domain.Playthrough(
```

Comment:
Consider moving this common setup code into a helper method, something like:

```python
def _get_valid_early_quit_playthrough(self):
  playthrough = stats_domain.Playthrough(...)
  playthrough.validate()
  return playthrough
```

Reply:
Done